app/statemachine.js

```diff
@@ -96,6 +96,7 @@ CoreStateMachine.prototype.getState = function () {
       enableSkip: this.volatileState.enableSkip,
       consume: false,
       volume: this.currentVolume,
+      deviceVolume: this.currentDeviceVolume,
```

Comment:
OK

Reply:
Sorry, I still see deviceVolume...

Reply:
I missed this because it has to do with the frontend, which I did not get any feedbacks at the time. Anyway, I made the change and pushed. Please review.